app/src/ui.cpp

```diff
@@ -79,6 +80,8 @@ void UI::showMainWindow()
     // view
     m_view->rootContext()->setContextProperty("daemon", m_daemon);
     m_view->rootContext()->setContextProperty("ui", this);
+    m_view->rootContext()->setContextProperty("_keyboardLayout", m_keyboardLayoutProvider);
+    qmlRegisterType<KeyboardLayoutProvider>("com.meego.maliitquick", 1, 0, "Layout");
```

Comment:
Please do not reuse others namespaces. Please move the code to registerQmlTypes() in sailfishconnect.cpp.
```suggestion
    qmlRegisterType<KeyboardLayoutProvider>("SailfishConnect.RemoteKeyboard", 0, 4, "KeyboardLayout");
```